index.html

```diff
@@ -476,11 +476,11 @@ <h2>Glossary</h2>
+<p><dfn class="lint-ignore export">User-supplied value</dfn>. Unreserved [=syntactic content=] in a [=vocabulary=] that is assigned by users, as distinct from reserved keywords in a given format or protocol. Users generally expect that their user-supplied values can be words or phrases in their preferred [=natural language=]. This is why [[CHARMOD]] recommends that "Specifications <em class="rfc2119">SHOULD NOT</em> arbitrarily exclude code points from the full range of Unicode code points from <span class="uname">U+0000</span> to <span class="uname">U+10FFFF</span> inclusive." [[CHARMOD-NORM]] gives <a href="https://www.w3.org/TR/charmod-norm/#dfn-user-supplied-value">some examples</a>. </p>
 
 <p><dfn class="lint-ignore">UTF-8</dfn>. <a href="https://unicode.org/glossary/#UTF_8">Unicode definition</a>: <q>A multibyte encoding for text that represents each Unicode character with 1 to 4 bytes, and which is backward-compatible with ASCII. UTF-8 is the predominant form of [=Unicode=] in web pages. More technically: (1) The <a href="https://unicode.org/glossary/#utf_8_encoding_form"> UTF-8 encoding form</a>. (2) The <a href="https://unicode.org/glossary/#utf_8_encoding_scheme"> UTF-8 encoding scheme</a>. (3) “UCS Transformation Format 8,” defined in Annex D of ISO/IEC 10646:2003, technically equivalent to the definitions in the Unicode Standard.</q></p>
 
-<p><dfn class="lint-ignore">UTF-16</dfn>. <a href="https://unicode.org/glossary/#UTF_16">Unicode definition</a>: <q>A multibyte encoding for text that represents each Unicode character with 2 or 4 bytes; it is not backward-compatible with ASCII. It is the internal form of [=Unicode=] in many programming languages, such as Java, C#, and JavaScript, and in many operating systems. More technically: (1) The <a href="https://unicode.org/glossary/#utf_16_encoding_form">UTF-16 encoding form</a>. (2) The <a href="https://unicode.org/glossary/#utf_16_encoding_scheme"> UTF-16 encoding scheme</a>. (3) “Transformation format for 16 planes of Group 00,” defined in Annex C of ISO/IEC 10646:2003; technically equivalent to the definitions in the Unicode Standard.</q></p>
+<p><dfn data-lt="UTF-16" class="lint-ignore">UTF-16</dfn>. <a href="https://unicode.org/glossary/#UTF_16">Unicode definition</a>: <q>A multibyte encoding for text that represents each Unicode character with 2 or 4 bytes; it is not backward-compatible with ASCII. It is the internal form of [=Unicode=] in many programming languages, such as Java, C#, and JavaScript, and in many operating systems. More technically: (1) The <a href="https://unicode.org/glossary/#utf_16_encoding_form">UTF-16 encoding form</a>. (2) The <a href="https://unicode.org/glossary/#utf_16_encoding_scheme"> UTF-16 encoding scheme</a>. (3) “Transformation format for 16 planes of Group 00,” defined in Annex C of ISO/IEC 10646:2003; technically equivalent to the definitions in the Unicode Standard.</q></p>
```

Comment:
Why do we need to add `data-lt="UTF-16"` here? I think `data-lt` is for defining *alternative* terms for a definition.